packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -76,18 +76,17 @@ const displayAndSetDefaults = async (context: $TSContext, projectPath: string, p
   context.print.success('The following configuration will be applied:');
   context.print.info('');
 
-  await displayConfigurationDefaults(context, defaultProjectName, defaultEnv, defaultEditorName);
+  displayConfigurationDefaults(context, defaultProjectName, defaultEnv, defaultEditorName);
```

Comment:
and not awaiting here?

Reply:
That await was unnecessary, the function is not async